### ISSUE TITLE
If the server only allows one connection, we get a lot of 421 errors

### ISSUE THREAD

**Issue by git-commit** [albertz/RandomFtpGrabber#2]

**Comment by albertz** [comment]
Yes. We could extend the sources.txt by some config about this. Or maybe
better: automatically parse that information from the message. And then
just avoid having too much connections to such server.



Am 29.12.2017 07:32 schrieb "Maximilian Berger" <notifications@github.com>:
